src/plugins/intel_gpu/tests/test_cases/border_gpu_test.cpp

```diff
@@ -6,6 +6,7 @@
 
 #include <intel_gpu/primitives/input_layout.hpp>
 #include <intel_gpu/primitives/border.hpp>
+#include "ngraph/runtime/reference/pad.hpp"
```

Comment:
I don't know it is right to use ngraph runtime in cldnn unit tests. border  layer only has ref kernel. So, I think it can move to gpu functional test. What do you think?

Reply:
Agree. It would be better to compare against planar format result.

Reply:
Done.

Reply:
Some unit test failing.

Reply:
Fixed